hello-neon/README.md

```diff
@@ -0,0 +1,61 @@
+Hello Neon
+==========
+Hello Neon is an Android sample that uses arm7 neon for fir filter implementation:
```

Comment:
'ARM NEON' as it's trademarked.
http://www.arm.com/products/processors/technologies/neon.php